app/components/ViewContainer.js

```diff
@@ -75,6 +75,8 @@ export default class ViewContainer extends Component {
       { this.state.view === 'list' ? <PinList
```

Comment:
Must have been introduced in a previous commit, but `<PinList />` also needs to be passed `updateRecent` and `updatePins`